typescript/src/schema/proto.ts

```diff
@@ -159,6 +161,26 @@ function longRunningMetadataType(
   );
 }
 
+// Detect non-AIP-compliant LRO method for DIREGAPIC, where the response type is customized Operation and the method is not polling method.
+// (TODO: summerji) Update the detector when DIREGAPIC LRO annotation merged in googleapis-discovery.
+function isDiregapicLRO(
+  packageName: string,
+  method: MethodDescriptorProto,
+  isDiregapic?: boolean
+): boolean {
+  const operationOutputType = toFullyQualifiedName(packageName, 'Operation');
+  return isDiregapic &&
+    method.outputType &&
+    method.outputType === operationOutputType &&
+    method.options?.['.google.api.http'] &&
+    !(
+      method.options?.['.google.api.http'].get ||
+      (method.name === 'Wait' && method.options?.['.google.api.http'].post)
```

Comment:
Are we saying that any POST method will be long running, unless it's of type `Wait`?

Reply:
Yes